Avoid warning

```
Library/PackageCache/dev.monry.upm.eventconnector@0.0.2-preview.3/Scripts/Connector/AudioEvent.cs(11,49): warning CS0649: Field 'AudioEvent.audioEventType' is never assigned to, and will always have its default value 
Library/PackageCache/dev.monry.upm.eventconnector@0.0.2-preview.3/Scripts/Connector/AudioController.cs(10,53): warning CS0649: Field 'AudioController.audioControlMethod' is never assigned to, and will always have its default value 
```

--- a/Assets/Scripts/Connector/AudioController.cs
+++ b/Assets/Scripts/Connector/AudioController.cs
@@ -7,7 +7,7 @@ namespace EventConnector.Connector
 {
     public class AudioController : EventConnector
     {
-        [SerializeField] private AudioControlMethod audioControlMethod;
+        [SerializeField] private AudioControlMethod audioControlMethod = default;
         [SerializeField]
         [Tooltip("If you do not specify it will be obtained by GameObject.GetComponent<AudioSource>()")]
         private AudioSource audioSource = default;

--- a/Assets/Scripts/Connector/AudioEvent.cs
+++ b/Assets/Scripts/Connector/AudioEvent.cs
@@ -8,7 +8,7 @@ namespace EventConnector.Connector
     [RequireComponent(typeof(AudioSource))]
     public class AudioEvent : EventConnector
     {
-        [SerializeField] private AudioEventType audioEventType;
+        [SerializeField] private AudioEventType audioEventType = default;
         [SerializeField]
         [Tooltip("If you do not specify it will be obtained by GameObject.GetComponent<AudioSource>()")]
         private AudioSource audioSource = default;

